api/resolvers/user.js

```diff
@@ -773,6 +767,12 @@ export default {
       })
 
       return !!subscription
+    },
+    csvRequest: async (user) => {
+      return user.csvRequest
+    },
+    csvRequestStatus: async (user) => {
+      return user.csvRequestStatus
```

Comment:
I think resolvers like these are obsolete. GraphQL automatically resolves fields of the parent object with same names.